test/knot-data/knot.conf

```diff
@@ -0,0 +1,116 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+server:
+    rundir: "/run/knot"
+    user: knot:knot
+    listen: 1.2.3.1@53
```

Comment:
Please use some reserved addresses. Address 1.1.1.1 provides public resolver, this uses its network. I think even 127.0.0.3 might be fully working for the test.

Reply:
Indeed. Maybe just use something from the range 10.0.0.0/8?